aws-ssm-document/src/main/java/com/amazonaws/ssm/document/CreateHandler.java

```diff
@@ -105,15 +105,20 @@ public CreateHandler() {
 
         final ResourceInformation resourceInformation = progressResponse.getResourceInformation();
 
+        final OperationStatus operationStatus = getOperationStatus(resourceInformation.getStatus());
         return ProgressEvent.<ResourceModel, CallbackContext>builder()
                 .resourceModel(resourceInformation.getResourceModel())
-                .status(getOperationStatus(resourceInformation.getStatus()))
+                .status(operationStatus)
                 .message(resourceInformation.getStatusInformation())
                 .callbackContext(progressResponse.getCallbackContext())
-                .callbackDelaySeconds(CALLBACK_DELAY_SECONDS)
+                .callbackDelaySeconds(setCallbackDelay(operationStatus))
                 .build();
     }
 
+    private int setCallbackDelay(final OperationStatus operationStatus) {
```

Comment:
nit: the "set" in the name indicates the return type is "void". we would want to name it as "getCallbackDelay" or "constructCallbackDelay" instead

Reply:
that makes sense, will update it